novuhq/website
# Echo page

This pull request brings echo page.

Steps to check:

- open [preview](https://deploy-preview-167--novu-website.netlify.app/echo/)
- make sure that everything looks and works as expected
